python/ray/train/_checkpoint.py

```diff
@@ -58,7 +58,24 @@ class Checkpoint(metaclass=_CheckpointMetaClass):
     Access the checkpoint contents locally using ``checkpoint.to_directory()``
     or ``checkpoint.as_directory``.
 
-    Example creating a checkpoint using ``Checkpoint.from_directory``:
+    Attributes
+    ----------
```

Comment:
Are they duplicated with the args in the `__init__()` function?

Reply:
Actually the init isn't shown automatically from [this API page](https://anyscale-ray--42422.com.readthedocs.build/en/42422/train/api/doc/ray.train.Checkpoint.html#ray.train.Checkpoint), so I think it's still useful.

I was following a [pandas example](https://github.com/pandas-dev/pandas/blob/736868671044605b0f2c2cc23ab5ea09fbc9a2ac/pandas/core/frame.py#L2543).